### PR TITLE
Logging and english

### PR DESCRIPTION
**Pull request by magamig**
Logging option and english error messages added.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "leaflet-export",
     "version": "0.2.0",
-    "homepage": "https://github.com/Flexberry/Leaflet.Export",
+    "homepage": "https://github.com/magamig/Leaflet.Export",
```
Comment (bratchikov) [inline bower.json]: magamig -> Flexberry

File: package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "name": "leaflet.export",
+  "version": "1.0.0",
+  "description": "Additional methods to L.map class provides export and print maps",
+  "main": "leaflet_export.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/magamig/Leaflet.Export.git"
```
Comment (bratchikov) [inline package.json]: magamig -> Flexberry

File: package.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/magamig/Leaflet.Export/issues"
```
Comment (bratchikov) [inline package.json]: magamig -> Flexberry

File: package.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  },
+  "homepage": "https://github.com/magamig/Leaflet.Export#readme",
```
Comment (bratchikov) [inline package.json]: magamig -> Flexberry

File: package.json [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  "main": "leaflet_export.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/magamig/Leaflet.Export.git"
+  },
+  "author": "",
+  "license": "ISC",
```
Comment (bratchikov) [inline package.json]: MIT

File: package.json [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  "description": "Additional methods to L.map class provides export and print maps",
+  "main": "leaflet_export.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/magamig/Leaflet.Export.git"
+  },
+  "author": "",
```
Comment (bratchikov) [inline package.json]: Flexberry Team, community

**Comment by magamig** [comment]
Requested changes 'committed'

**Comment by magamig** [comment]
@bratchikov bump

**Pull request merged**